### ISSUE TITLE
MD029 is triggered on any multiline ordered list items.

### ISSUE THREAD

**Issue by rroman81** [DavidAnson/markdownlint#45]
I am using the markdownlint v0.6.2 and I am seeing that the MD029 rule is always triggered even when using the sample provided inside the rule.
i.e.
```markdown
1. Do this.
2. Do that.
3. Done.
```
should pass the issue but it is generating green swiggle with MD029 tooltip.

**Comment by DavidAnson** [comment]
Documentation: https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md029---ordered-list-item-prefix

MD029 has two modes: `one` and `ordered`. The default mode is `one` which looks for the following style:

```
1. First
1. Second
1. Third
```

The benefit of this style is that lines can be reordered without having to worry about the numbers going out of sequence. If you wish to change the style to `ordered` per your example above, you can update the rule configuration via one of the supported mechanisms:

- https://github.com/DavidAnson/markdownlint#configuration
- https://github.com/DavidAnson/vscode-markdownlint#configure

**Comment by s10wen** [comment]
Landed here as this caught me out as well, updating to `ordered` it is then, thanks.

**Comment by s10wen** [comment]
I'm new to VS, I've gone to settings and added the markdown code, but having trouble with the syntax, tried a few things but no luck, does anyone have an example for the settings code I should be using here please?

e.g.

```
{
    "workbench.colorTheme": "Monokai",
    "workbench.iconTheme": "vscode-icons",
    "editor.renderWhitespace": "all",
    "[markdown]": {
      "editor.wordWrap": "on",
      "editor.quickSuggestions": false,
      "editor.ol-prefix": "ordered"
    }
  }
```

**Comment by DavidAnson** [comment]
On a phone right now, but if you edit VS Code confit files from within the app, it will provide auto-complete suggestions to help. You should end up with something like what you see towards the bottom of this section:
https://github.com/DavidAnson/vscode-markdownlint/blob/master/README.md#configure

**Comment by s10wen** [comment]
Thanks, managed to sort this, in case anyone else comes here, I'd clicked the top one here, instead of the bottom example code:

<img width="500" alt="screen shot 2017-09-27 at 08 11 01" src="https://user-images.githubusercontent.com/1136114/30900308-78376206-a35b-11e7-814d-72875274c870.png">

I now have:

```
{
    "workbench.colorTheme": "Monokai",
    "workbench.iconTheme": "vscode-icons",
    "editor.renderWhitespace": "all",
    "markdownlint.config": {
      "MD029": false,
      "MD034": false
    }
  }
```

**Comment by amoghskulkarni** [comment]
I created `.markdownlint.json` in my root folder and added following snippet in that -

```json
{
    "default": true,
    "ol-prefix": { "style": "ordered" }
}
```

But the linting still occurs at `2.` saying `MD029/ol-prefix: Ordered list item prefix [Expected: 1; Actual: 2]`. 

Am I missing something?

**Comment by DavidAnson** [comment]
Can you show the Markdown you are getting the wrong result for?

**Comment by DavidAnson** [comment]
And say what tool or code you are using to lint.

**Comment by ScottWorks** [comment]
In case you're confused about how to use ordered lists with this rule in place...

> It’s important to note that the actual numbers you use to mark the list have no effect on the HTML output Markdown produces.

Original Markdown Spec
